gulpfile.js

```diff
@@ -555,11 +555,19 @@ function createTestSource(testsName, { bot = false, xfaOnly = false } = {}) {
     console.log("### Running " + testsName + " tests");
 
     const PDF_TEST = process.env.PDF_TEST || "test_manifest.json";
+    let forceNoChrome = false;
     const args = ["test.js"];
     switch (testsName) {
       case "browser":
         if (!bot) {
           args.push("--reftest");
+        } else {
+          const os = process.env.OS;
+          if (/windows/i.test(os)) {
+            // The browser-tests are too slow in Google Chrome on the Windows
+            // bot, causing a timeout, hence disabling them for now.
```

Comment:
I was just thinking: maybe it's good to check the Puppeteer upgrade in a separate PR to make sure it's isolated from the other changes w.r.t. `structuredClone` here? It could be that the Puppeteer upgrade itself is fine, but we're hitting another bug on Windows, possibly because `structuredClone` is quite new in Chrome and in still in beta?

Reply:
> maybe it's good to check the Puppeteer upgrade in a separate PR to make sure it's isolated from the other changes w.r.t. `structuredClone` here?

Sure, I suppose that I can split that out into its own PR and run tests.

> but we're hitting another bug on Windows, possibly because `structuredClone` is quite new in Chrome and in still in beta?

I'd be quite surprised if there's platform-specific bugs related to `structuredClone`, and note that everything works fine in Google Chrome on Linux.
Furthermore, the only way that we'd even invoke `structuredClone` in the browser-tests is if workers are somehow *disabled*, since that code won't even run otherwise.

Reply:
As I suspected, we get the same problems in PR #14471 as well; hence this is *not* connected to the `structuredClone` code.

---

While I haven't run the *entire* browser-test suite locally in Chrome, I cannot reproduce the more-or-less immediate slowdown that can be observed in the bot logs (where Google Chrome is "falling behind" almost instantly).